README.rst

```diff
@@ -40,31 +40,27 @@ Next step is to install assets (js, css etc.):
     plupload
     ...
 
-Now you can follow the standard invenio installation and development procedures, such as running `inveniomanage collect`.
+Now you can follow the standard Invenio installation and development procedures, such as running `inveniomanage collect`.
 
 ---------------------
 Demosite installation
 ---------------------
-As INSPIRE uses `Mixer <https://github.com/klen/mixer>`_ to load database fixtures, you should install the library:
+Since the database was already initialized while installing Invenio, you run:
 
 .. code-block:: bash
 
-     $ pip install -e .[mixer]
+     $ inveniomanage database recreate
```

Comment:
I am not sure about this. In the instructions https://github.com/inveniosoftware/invenio/blob/pu/INSTALL.rst#33-development the

``` $ inveniomanage database create```

happens once both Invenio and the demosite have been installed. Since INSPIRE replaces the demosite but the instructions are the same I assume the database should not be created at this stage and ```$ inveniomanage database create``` is the right command?